hashintel/hash
# H-203: Update Realtime service, watch for changes to Linear types

## 🌟 What is the purpose of this PR?

This PR lays the groundwork for syncing changes to Linear-related entity types in HASH back to the Linear API.

Specifically:
1. Update the Realtime service for the new database tables (the previous implementation predated the Graph API)
2. Add a subscription to the Realtime service in the API, and detect changes to Linear-related entities in HASH

It also removes some defunct code related to the previous Collab realtime subscription. 

## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:


### 📜 Does this require a change to the docs?




The changes in this PR:


### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:



## 🐾 Next steps

- Actually do something in the Linear API based on what type of entity has changed and how

## ❓ How to test this?

1.  Create or update an entity of the Linear issue type
1.  Confirm that the 'Linear issue change detected' log appears
